content/en/integrations/guide/source-code-integration.md

```diff
@@ -31,84 +35,21 @@ Datadog Agent v7.35.0 or later is required.
 
 If you have [APM][6] set up already, navigate to [**Integrations** > **Link Source Code**][7] and configure the source code integration for your backend services.
 
-## Link active commits
+## Tag your telemetry with Git information
```

Comment:
@drichards-87 In this part, we want customers to perform *either* the `Embed Git information in your build artifacts` OR the `Configure telemetry tagging` part. Do you think it's clear enough as it is ? I'm afraid readers might carry on after doing the first one and do both. Do you have any recommendation to make it even clearer ?

Reply:
Reading through the section again, I do think it's clear. That said, if we want to make it even more explicit, I would update the first sentence in the [Configure telemetry tagging](https://docs-staging.datadoghq.com/ronan/sci-improvements/integrations/guide/source-code-integration/?tab=go#configure-telemetry-tagging) section. For example:

> For unsupported languages, use the git.commit.sha and git.repository_url tags to link data to a specific commit.

Reply:
Thanks, I added that ! I think it should be ready to merge now.